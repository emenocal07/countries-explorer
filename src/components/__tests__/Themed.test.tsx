import { render } from '@testing-library/react-native'
import React from 'react'
import { Text, View } from '../Themed'

describe('Themed', () => {
  it('renders Text without crashing', () => {
    const { toJSON } = render(<Text>Test</Text>)
    expect(toJSON()).toMatchSnapshot()
  })
  it('renders View without crashing', () => {
    const { toJSON } = render(
      <View>
        <Text>Test</Text>
      </View>
    )
    expect(toJSON()).toMatchSnapshot()
  })
})
