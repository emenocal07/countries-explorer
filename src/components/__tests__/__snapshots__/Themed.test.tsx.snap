// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Themed renders Text without crashing 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Test
</Text>
`;

exports[`Themed renders View without crashing 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#000",
        },
        undefined,
      ]
    }
  >
    Test
  </Text>
</View>
`;
